lib/propolis/src/hw/nvme/bits.rs

```diff
@@ -867,7 +867,7 @@ pub struct IdentifyController {
     /// See NVMe 1.0e Section 4.2, Figure 8 Command Format - Admin and NVM Vendor Specific Commands (Optional)
     pub avscc: u8,
     /// Reserved
-    pub _resv2: [u8; 246],
+    pub _resv2: [u8; 247],
```

Comment:
Would a `sizeof` check on the struct have caught this?  If so, does it make sense adding a unit test with that?

Reply:
There was one, and it didn't catch it because of the final alignment padding. Adding `packed(1)` to the struct definition makes the test fail on master, and pass on this branch:

```
running 1 test
test hw::nvme::bits::test::entry_sizing ... FAILED

failures:

---- hw::nvme::bits::test::entry_sizing stdout ----
thread 'hw::nvme::bits::test::entry_sizing' panicked at 'assertion failed: `(left == right)`
  left: `4095`,
 right: `4096`', lib/propolis/src/hw/nvme/bits.rs:1145:9
```

I'll add this to all of these structs.